### PR TITLE
Typescript migration seeders

### PR DESCRIPTION
**Pull request by ArturFedorov**
Только название содержанию не соответствует

### CODE REVIEW

File: server/config/config.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "development": {
     "dialect": "postgres",
     "username": "postgres",
-    "password": null,
+    "password": "123",
```
Comment (Igogrek) [inline server/config/config.json]: Nope

File: .gitignore [hunk 2]
```diff
@@ -37,3 +37,5 @@ testem.log
 # System Files
 .DS_Store
 Thumbs.db
+
+/server/config/
```
Comment (Igogrek) [inline .gitignore]: Revert gitignore -not needed

File: server/features/project/project.service.ts [hunk 3]
```diff
@@ -2,7 +2,8 @@ import {Project} from '../../models/Project';
 import {Scopes} from '../../sequelize/Scopes';
 import {Technology} from '../../models/Technology';
 import {Schedule} from '../../models/Schedule';
-import {sequelize} from '../../sequelize/sequelize';
+import {ProjectTechnology} from '../../models/ProjectTechnology';
+import sequelize from '../../sequelize/sequelize';
```
Comment (Igogrek) [inline server/features/project/project.service.ts]: sequalize is default export?

File: server/features/project/project.service.ts [hunk 4]
```diff
@@ -70,14 +71,15 @@ const projectService = {
...
-  }, { include: [Schedule ]})
+  })
     .then(projectNew => {
-      const technologies = parse.parseTechnology(project.technologies);
-      const instances = technologies.map(tech => Technology.build(tech));
-      projectNew.$set('technologies', instances);
-
-      return projectNew;
+      const schedules = parse.parseShedules(projectNew, project.schedules);
+      const instances = parse.parseTechnology(project.technologies);
```
Comment (Igogrek) [inline server/features/project/project.service.ts]: Variables are not needed, just do this inside Promise.all

File: server/server-addons/server.constants.ts [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+ * @enum {number}
+ */
+export enum ServerConstants {
+    API_PREFIX = '/api',
+    DEFAULT_PORT = '3000',
+    DEFAULT_INDEX_PATH = 'dist/index.html',
+    DEFAULT_STATIC_PATH = 'dist',
+    IMAGES_ROUTE = '/server/images',
+    IMAGES_PATH = 'server/images',
+    PORT_ALLIAS = 'port',
```
Comment (Igogrek) [inline server/server-addons/server.constants.ts]: typo:ALIAS. Is it needed at all?

Comment (ArturFedorov) [reply server/server-addons/server.constants.ts]: Need at all? whole Enum or PORT_ALIAS

File: server/sequelize/sequelize.ts [hunk 6]
```diff
@@ -6,4 +6,4 @@ const env = process.env.NODE_ENV || 'development';
 const sequelize = new Sequelize(config[env]);
 sequelize.addModels([`${__dirname}/../models`]);
 
-export default sequelize;
+export {sequelize};
```
Comment (Igogrek) [inline server/sequelize/sequelize.ts]: Revert - I only asked a question. default export is good in this case

**Pull request merged**